service/certconfig/v2/error.go

```diff
@@ -0,0 +1,12 @@
+package v2
```

Comment:
There is now the v2 certconfig package. Note that the `vaultpkiv1` resource got moved to here as well.